.github/workflows/test.yml

```diff
@@ -27,6 +28,25 @@ jobs:
   #                 fi
   #             fi
   #         done
+=======
+  unreleased-deps:
```

Comment:
@nobuQuartile @AungKoKoLin1997 Can you explain the implications of enabling this job? I forget how it was disabled.

Reply:
@yostashiro @nobuQuartile My mistake. Just disable that CI because that CI will fail if there is any `/` in `test-requirements.txt`.
Since OCA repos are related each other, they don't have any kind of link inside this file but we have to put dependency modules from another repos in this file. So, this CI will always red.

Reply:
Okay, thanks for the explanation!